company-profiles/delighted.md

```diff
@@ -6,25 +6,60 @@ Delighted is a company which provides software that can gather actionable feedba
 
 ## Company size
 
-10-20 Employees
+100-200
 
-## Remote status
+## Region
```

Comment:
Removed the markdown from the regions. 

It should be 

## Region 

* USA
* Asia
* Europe